docs/devGuide/settingUp.md

```diff
@@ -45,13 +49,16 @@ We recommend the **WebStorm IDE** for working with MarkBind code.
 
 1. **Congratulations!** Now you are ready to start modifying MarkBind code.
 
+
+## Setting up the git hooks (optional but recommended)
+
+1. **Install git hooks** by running `npm run install:hooks` in the **root folder** of your cloned repo.
```

Comment:
It may be better to move and document the commands here directly as `python` vs `python3` alias availability is very system setup dependent.

```suggestion
There are a few [Git hooks](./workflow.md#git-hooks) implemented using the [precommit](https://pre-commit.com/) tool that runs common tasks like linting, automated tests and compiling typescript files.

**Install the hooks** by running `python3 ./pre-commit/pre-commit-2.20.0.pyz install` in the **root folder** of your cloned repo.

If you ever need to uninstall the hooks, simply run `uninstall` instead of `install`.
```